dynamic-plugins/wrappers/roadiehq-scaffolder-backend-module-http-request/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "roadiehq-scaffolder-backend-module-http-request",
+  "version": "4.1.10",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin"
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @roadiehq/scaffolder-backend-module-http-request --override-interop default"
+  },
+  "dependencies": {
+    "@backstage/backend-dynamic-feature-service": "^0.2.2",
+    "@backstage/backend-plugin-api": "0.6.13",
```

Comment:
Wouldn't it be possible to use on of the dependency versions already present in the `yarn.lock`: https://github.com/janus-idp/backstage-showcase/pull/1207/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR2481 ?